common/default_chains_mock_mainnet.go

```diff
@@ -0,0 +1,53 @@
+//go:build MOCK_MAINNET
+// +build MOCK_MAINNET
+
+package common
+
+func EthChain() Chain {
+	return Chain{
+		ChainName: ChainName_eth_mainnet,
+		ChainId:   1,
+	}
+}
+
+func BscMainnetChain() Chain {
+	return Chain{
+		ChainName: ChainName_bsc_mainnet,
+		ChainId:   56,
+	}
+}
+
+func ZetaChain() Chain {
+	return Chain{
+		ChainName: ChainName_zeta_mainnet,
+		ChainId:   70000,
+	}
+}
+
+func BtcMainnetChain() Chain {
+	return Chain{
+		ChainName: ChainName_btc_mainnet,
+		ChainId:   8332,
+	}
+}
+
+func PolygonChain() Chain {
+	return Chain{
+		ChainName: ChainName_polygon_mainnet,
+		ChainId:   137,
+	}
+}
+
+func DefaultChainsList() []*Chain {
+	chains := []Chain{
+		BtcMainnetChain(),
+		BscMainnetChain(),
+		EthChain(),
+		ZetaChain(),
```

Comment:
i don't think this is used anywhere, having it here is confusing

Reply:
The value is used for this. 
https://github.com/zeta-chain/zeta-node/blob/8a842c0147927b5db627457effd363b1bb8ba475/zetaclient/evm_client.go#L695-L699

We can defeinitly remove it, but it should be done in a separate pr; this is for adding the config only.